charts/matrix-server/values.yaml

```diff
@@ -16,7 +16,7 @@ db:
   user: synapse_user
   password: password
   name: synapse
-  host: postgresql-postgresql
+  host: postgresql
```

Comment:
so with the change from the w3f to the bitnami chart that is the new naming convention, right ?

Reply:
It seems so, yes. The postgres-master/slave now is called postgres-primary/read and the service name changed from postgresql-postgresql to postgresql only.